src/client.zig

```diff
@@ -195,14 +229,22 @@ pub const Client = struct {
         }
 
         if (reply.header.request < queued_request.message.header.request) {
-            log.debug("{} on_reply: Dropping duplicate message. request_number_min={}", .{ self.id, self.request_number_min });
+            log.debug(
+                "{} on_reply: Dropping duplicate message. request_number_min={}",
```

Comment:
For the person reading this debug log, could we indicate more how `request_number_min` relates to the reply we are dropping? Perhaps we could also refer to `request_number_min` here in the log as `request` since this is the `Header.request` number of the reply we are dropping, whereas `request_number_min` is more our internal implementation?